runtime/parachains/src/paras_inherent/mod.rs

```diff
@@ -599,12 +624,15 @@ impl<T: Config> Pallet<T> {
 
 		// Assure the maximum block weight is adhered.
 		let max_block_weight = <T as frame_system::Config>::BlockWeights::get().max_block;
+		// TODO subtract enactment weight - return sanitized bitfields from enactment
```

Comment:
There has to be additional logic, when selecting a `availablity_bitfield`, if that has consequences regarding weight consumed.

Reply:
The comment is a bit stale; my solution to this was to take into account the worst case enactment wheight whenever we include the bitfields.

Reply:
Hence why I changed it to

```
	let total = total_bitfields_weight
		.saturating_add(total_candidates_weight)
		.saturating_add(total_enact_candidates_weight);
```

Reply:
I see.